languages/python/reference/README.md

```diff
@@ -538,3 +538,181 @@ These will very rarely be encountered in the wild, the first because it's more o
 [variables]: ../../../reference/concepts/variables.md
 [while-loops]: ./concepts/while_loops.md
 [zen-of-python]: ./concepts/zen_of_python.md
+
+## Implementations  
+
+
+| Concept Name | Issue Link | PR Link |
+|--------------|:----------:|--------:|
+| [and][keyword-and] | [ ]() | [ ]() |
+| [Anonymous functions][anonymous-functions-general] `functional`| [ ]() | [ ]() |
+| [Arithmetic][arithmetic-general]| [ ]() | [ ]() |
+| [`*args`][star-args] | [ ]() | [ ]() |
+| [Arguments & Parameters][arguments-and-parameters]| [ ]() | [ ]() |
+| [Argument unpacking][argument-unpacking]| [ ]() | [ ]() |
+|[as][keyword-as]|[ ]() | [ ]() |
+|[assert][keyword-assert]|[ ]() | [ ]() |
+| [Assignment][assignment] | [ ]() | [ ]() |
+| [Binary numbers][binary-numbers]| [ ]() | [ ]() |
+| [bitflags][bitflags]| [ ]() | [ ]() |
+| [Bitwise manipulation][bitwise-manipulation-general]| [ ]() | [ ]() |
+| [Bitwise operators][bitwise-operators]| [ ]() | [ ]() |
+| [bool][builtin-types-bool] `basic`|  [#1030](https://github.com/exercism/v3/issues/1030)         |         |
+| [Boolean logic][boolean-logic-general]| [ ]() | [ ]() |
+| [Boolean operators][boolean-operators]| [ ]() | [ ]() |
+| [Boolean values][boolean-values]| [ ]() | [ ]() |
+| [Booleans are integers][booleans-are-integers]| [ ]() | [ ]() |
+| [Bracket notation][bracket-notation] | [ ]() | [ ]() |
+| [break][keyword-break] | [ ]() | [ ]() |
+| [bytearray][builtin-types-bytearray] `intermediate`| [ ]() | [ ]() |
+| [bytes][builtin-types-bytes] `intermediate`| [ ]() | [ ]() |
+| [Call semantics][call-semantics]| [ ]() | [ ]() |
+| [Class][keyword-class] | [ ]() | [ ]() |
+| [Classes][classes-general] | [ ]() | [ ]() |
+| [Class Composition][class-composition] | [ ]() | [ ]() |
+| [Class Inheritance][class-inheritance] | [ ]() | [ ]() | 
+| [Class members][class-members]| [ ]() | [ ]() |
+| [Class methods][class-methods]| [ ]() | [ ]() |
+| [Comments][comments-general]| [ ]() | [ ]() |
+| [Comparison operators][comparison-operators]| [ ]() | [ ]() |
+| [complex][builtin-types-complex] `intermediate` | [ ]() | [ ]() |
+| [Composition][composition-general] | [ ]() | [ ]() |
+| [Comprehension Syntax][comprehension-syntax]| [ ]() | [ ]() |
+| [Conditionals][conditionals-general] | [ ]() | [ ]() |
+| [Constructor][constructor] | [ ]() | [ ]() |
+| [Conditional structures][conditional-structures] | [ ]() | [ ]() |
+| [Constants][constants] | [ ]() | [ ]() |
+|[continue][keyword-continue]| [ ]() | [ ]() |
+| [Custom Classes][custom-classes] | [ ]() | [ ]() |
+| [decorators][decorators] `functional` | [ ]() | [ ]() |
+| [def][keyword-def] | [ ]() | [ ]() |
+| [Default arguments][default-arguments] | [ ]() | [ ]() |
+|[del][keyword-del] |  [ ]() | [ ]() |
+| [dict][builtin-types-dict] `basic`|   [#877](https://github.com/exercism/v3/issues/877)  |         |
+| [Dict comprehension][dict-comprehension] |  [ ]() | [ ]() |
```

Comment:
There's a link reference for `dict-comprehension` here, but there's no corresponding link entry ... we'll need to add `[dict-comprehension]: ./concepts/dict_comprehension.md` in the appropriate location in the link entries.

Reply:
Added in entries for dict, list and set.  😄